app/templates/index.html

```diff
@@ -41,5 +41,55 @@
         </div>
         <h1>{{ title }}</h1>
     </div>
+
+    <div class="education-section">
+        <h2>Education</h2>
+        
+        <div class="member-tabs">
+            {% for member in members_education %}
+              <button id="tab-member-{{ loop.index }}" class="tab-button{% if loop.index == 1 %} active{% endif %}" onclick="openTab('member', '{{ loop.index }}')">
+                {{ member.name }}
+              </button>
+            {% endfor %}
+        </div>
+
+        <div class="timeline-content">
+            {% for member in members_education %}
+            <ul id="content-member-{{ loop.index }}" class="timeline{% if loop.index == 1 %} active{% endif %}">
+                {% for education in member.education %}
+                <li class="timeline-item">
+                    <div class="timeline-date">{{ education.date }}</div>
+                    <div class="timeline-education">
+                        <h3>{{ education.degree }}</h3>
+                        <p>{{ education.institution }}</p>
+                    </div>
+                </li>
+                {% endfor %}
+            </ul>
+            {% endfor %}
+        </div>
+    </div>
+
+    <script>
```

Comment:
Would it be better here to separate this into another js file and call that method? I would think so, but am asking as you might have another reason to do it like this

Reply:
I think separating it would definitely be better, I'll get to it right away!